src/main.js

```diff
@@ -1,44 +1,39 @@
-import { render } from './framework/render';
 import EventsModel from './model/events-model';
 import EventPresenter from './presenter/event-presenter';
+import FilterPresenter from './presenter/filter-presenter';
 import TripPresenter from './presenter/trip-presenter';
-import { generateFilter } from './utils/filter-event';
-import FilterView from './view/filter-view';
 
-const eventsModel = new EventsModel();
-
-const renderHeader = () => {
-  const headerContainer = document.querySelector(
-    '.page-header .page-header__container'
-  );
-  const tripControlElement = headerContainer.querySelector(
-    '.trip-controls__filters'
-  );
+const headerContainer = document.querySelector(
+  '.page-header .page-header__container'
```

Comment:
Этот селектор можно укоротить и в теории, он отработает немного быстрее

```suggestion
  '.page-header__container'
```